release-notes-3.7.1.md

```diff
@@ -1,8 +1,8 @@
-!not-ready-for-release!
-
 #### Version Number
 ${version-number}
 
 #### New Features
+- SCMOD-12531  Updated parent artifact to the latest version.
```

Comment:
Do we usually record such things?   There is no other change that I could add and it did not seem right to leave it with none.

Reply:
I normally go with this if a release only has dependency updates, maybe @michael-bryson can confirm?

```
#### Version Number
${version-number}

#### New Features
- None

#### Bug Fixes
- Dependency updates only in this release.

#### Known Issues
- None
```

Reply:
Ans just for your info, this is what I use for an initial release of a repo:

```
#### Version Number
${version-number}

#### New Features
- Initial release

#### Known Issues
- None
```

Reply:
I agree with @rorytorneymf, it is certainly not a New Feature and since this is a patch release there should not be any new features anyway.